spec/factories_spec.rb

```diff
@@ -0,0 +1,9 @@
+require 'rails_helper'
+
+FactoryGirl.factories.map(&:name).each do |factory_name|
+  describe "The #{factory_name} factory" do
+    it 'is valid' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.